### PR TITLE
Add changes to benchmarks to securely pass Cluster endpoint details

### PR DESCRIPTION
**Pull request by Divyaasm**
### Description
Changes to accommodate hiding the sensitive information related to user OS endpoint while triggering benchmark tests

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: jenkins/opensearch/benchmark-test-endpoint.jenkinsfile [hunk 1]
```diff
@@ -21,20 +21,29 @@ pipeline {
...
             name: 'CLUSTER_ENDPOINT',
             description: 'Provide an endpoint to a cluster for running benchmark tests against it.',
-            trim: true
         )
         booleanParam(
             name: 'SECURITY_ENABLED',
             description: 'Mention if the cluster is secured or insecured.',
             defaultValue: false,
         )
+        password(
```
Comment (rishabh6788) [inline jenkins/opensearch/benchmark-test-endpoint.jenkinsfile:36]: default this to `admin`.

File: run.sh [hunk 2]
```diff
@@ -20,5 +20,5 @@ echo "Installing dependencies in $DIR ..."
 export PIPENV_PIPFILE="$DIR/Pipfile"
 python3 -m pipenv install
 
-echo "Running "$1" ${@:2} ..."
+if [[ ! "$1" == *benchmark* ]]; then (echo "Running "$1" ${@:2} ..."; ); fi
```
Comment (rishabh6788) [inline run.sh:23]: Why this?

File: src/test_workflow/benchmark_test/benchmark_args.py [hunk 3]
```diff
@@ -133,6 +135,8 @@ def __init__(self) -> None:
         self.min_distribution = args.min_distribution
         self.component = args.component
         self.insecure = args.insecure
+        self.username = args.username
```
Comment (rishabh6788) [inline src/test_workflow/benchmark_test/benchmark_args.py:138]: define username and password vars like others.

File: run.sh (hunk 2 above)
Comment (Divyaasm) [reply run.sh:23]: `Running "$1" ${@:2} ...` This applies to all the workflows in the build library. Avoided the logging statements only for Benchmark workflow ti hide the sensitive information.

File: jenkins/opensearch/benchmark-test-endpoint.jenkinsfile (hunk 1 above)
Comment (Divyaasm) [reply jenkins/opensearch/benchmark-test-endpoint.jenkinsfile:36]: I have added that in python end. Can do the same on in Jenkins too.

**Review by rishabh6788 — APPROVED** [review]

**Pull request merged**